plug-ins/movement/recallmovement.cpp

```diff
@@ -120,23 +121,34 @@ bool RecallMovement::checkForsaken( Character *wch )
 
 bool RecallMovement::checkNorecall( Character *wch )
 {
-    if (IS_SET(from_room->room_flags, ROOM_NO_RECALL))
+    if (IS_SET(from_room->room_flags, ROOM_NO_RECALL)) {
+        wch->pecho("Эта местность проклята, здесь боги не услышат твою молитву о возвращении.");
```

Comment:
@fiorine if `checkNorecall` and `checkCurse` methods now always show a failure message, no need to duplicate it in `checkForsaken`.  Also we need to use `msgSelfParty` so that pet's failure to recall is always displayed to its master.